src/api/offers.js

```diff
@@ -114,7 +115,7 @@ export default ({offerService}: { offerService: OfferService }): Router => {
         if (!offer) {
           response.status(HttpStatus.NOT_FOUND).json('No such offer')
         } else if (offer.deleted) {
-          response.status(HttpStatus.BAD_requestUEST).json('Already deleted')
+          response.status(HttpStatus.BAD_REQUEST).json('Already deleted')
```

Comment:
I think GONE makes sense here: https://httpstatuses.com/410

Reply:
This wasn't part of this PR anyways, just saw it and corrected it :).
We will introduce proper errors in another task.

Reply:
Does this task already exist? If yes ignore this comment :)